pool/src/lib.rs

```diff
@@ -446,6 +488,56 @@ impl LendingPoolTrait for LendingPool {
         paused(&env)
     }
 
+    fn get_account_data(env: Env, who: Address) -> Result<(i128, i128, i128), Error> {
```

Comment:
Would it be more clear, if we rename it to `get_account_debt_status`?

Reply:
May be get_position